README.markdown

```diff
@@ -150,6 +150,11 @@ Supported options:
 
 (I.e. you want only interface declarations for your Protobuf types, then pass all three of `outputEncodeMethods`, `outputJsonMethods`, and `outputClientImpl` as `false`, i.e. `--ts_proto_opt=outputEncodeMethods=false,outputJsonMethods=false,outputClientImpl=false`.)
 
+The following options are very useful for NestJS and available if `outputEncodeMethods`, `outputJsonMethods`, and `outputClientImpl` are set as `false`
+
+* If you pass `--ts_proto_opt=useMetadata=true`, the last argument accepts grpc metadata
```

Comment:
Maybe `addGrpcMetadataParam`? or `addGrpcMetadata`?

Reply:
Sure, sounds logical - I will change it to addGrpcMetadata

Reply:
Hi @stephenh I have added the change to the parameter (addGrpcMetadata) and updated the readme.

I think that's more semantic right?